docs/site/DEVELOPING.md

```diff
@@ -510,3 +510,25 @@ configuration, it's important to verify that all usage scenarios keep working.
 5.  Test integration with supported IDEs:
 
     - [VS Code](./VSCODE.md#how-to-verify-tslint-setup)
+
+### tsconfig files
+
+In the [`loopback-next`](https://github.com/strongloop/loopback-next) monorepo,
+`TypeScript` is set up in two places:
+
+1. When using VS Code, the `TypeScript` engine views `loopback-next` as a single
+   big project.
+
+- This enables the "refactor - rename" command to change all places using the
```

Comment:
This content does not render well.

<img width="914" alt="screen shot 2019-01-08 at 16 14 58" src="https://user-images.githubusercontent.com/1140553/50839437-9939c900-1360-11e9-8609-dbde770ccdfe.png">

I am proposing to indent this paragraph with four spaces, so that the text stays inside the list item.

```markdown
1. When using VS Code, the `TypeScript` engine views `loopback-next` as a single
    big project.

    This enables the "refactor - rename" command to change all places using the
    renamed symbol, and also makes "go to definition" command jump to `.ts` files
    (etc.)
```